source/agora/consensus/EnrollmentManager.d

```diff
@@ -96,8 +96,8 @@ public class EnrollmentManager
     /// Random key for enrollment
     private Pair signature_noise;
 
-    /// Next height for pre-image revelation
-    private Height next_reveal_height;
+    /// Distance of the preimage being revealed next time
+    private uint next_reveal_dist;
```

Comment:
```suggestion
    private uint next_reveal_distance;
```

I prefer it longer like this because `dist` can mean other things. Like "distribution".